### DISCUSSION TITLE
[ Feature Request ] Tree View Panel when browsing open folder without press enter

### DISCUSSION THREAD

**Discussion by Augusto7743** [doublecmd/doublecmd#1148] [Suggestions]
In Tree View Panel when browsing folder to be listed in tabs need press enter.
If the user need browse several folders need press enter to each folder.
Total Commander have that feature to open folder automatically without pressing enter.

The feature to open folder not needing press enter plus a delay in mliseconds to automatically open the folder is need to add in Double Commander.

Have a nice week.

**Comment by roke-julian-lockhart** [comment]
Why is this a discussion now?

**Reply by Skif-off** [reply]
[UPD: Use GitHub Discussions to suggest a new feature or improvement](https://github.com/doublecmd/doublecmd/commit/ef08388b48fddec67ec5d852f51bddd1c55f1bfe).

**Reply by roke-julian-lockhart** [reply]
Ah, that's a shame.

**Reply by alexx2000** [reply]
No, it is optimization and ordering. User makes a suggestion, if developer accept it and want to implement then developer can convert it into issue and assign himself to it.
